### ISSUE TITLE
add debounce function to prevent quick repeated clicks

### ISSUE THREAD

**Issue by 490626721** [pratik227/vue3-json-excel#5]
**Is your feature request related to a problem? Please describe.**
Users may repeatedly download files by clicking the button quickly

**Describe the solution you'd like**
like: 
```
<download-excel :loading="downLoadLoading" :data="json_data" :fields="json_fields" worksheet="My Worksheet" name="filename.xls" >
  Download Excel
</download-excel>
```

**Describe alternatives you've considered**
Or handle this event internally through the API interface

**Comment by pratik227** [comment]
Hi @490626721,

Added new props called **debounce** type **Number**. Default value is **500**.

Example

`<json-excel class="btn btn-default"
                :data="json_data"
                :fields="json_fields" :debounce="3000"
                worksheet="My Worksheet"
                name="filename.xls">
      Download File
    </json-excel>
`

It's available in version 0.0.6.

Thanks,
Pratik Patel

**Comment by pratik227** [comment]
Closing this issue
